### PR TITLE
Fix: remove auth tokens from export

### LINKED ISSUE #8098: [BUG] export does not work anymore after the update from 2.13.0 to 2.13.1
### Description

Paperless export does not work anymore after the update from 2.13.0 to 2.13.1

### Steps to reproduce

docker compose -f /home/paperless/docker-compose.yml exec -T webserver document_exporter ../export --delete --passphrase XYZ

### Webserver logs

```bash
Traceback (most recent call last):
  File "/usr/src/paperless/src/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.12/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.12/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.12/site-packages/django/core/management/base.py", line 413, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.12/site-packages/django/core/management/base.py", line 459, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/management/commands/document_exporter.py", line 218, in handle
    self.dump()
  File "/usr/src/paperless/src/documents/management/commands/document_exporter.py", line 289, in dump
    self.encrypt_secret_fields(manifest_dict)
  File "/usr/src/paperless/src/documents/management/commands/document_exporter.py", line 568, in encrypt_secret_fields
    if manifest_record["fields"][field]:
       ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^
KeyError: 'key'
```

### Browser logs

_No response_

### Paperless-ngx version

2.13.1

### Host OS

Debian 12 x86_64

### Installation method

Docker - official image

### System status

_No response_

### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

### PR DESCRIPTION
**Pull request by shamoon**
## Proposed change

~~For some reason `serialize` doesn't work as expected for auth tokens. Options here are this PR or I can change it to just use "pk" as the crypt field key because that's what it serializes to (weird). Or we can just remove auth tokens from the export.~~

~~LMK~~

~~Currently `auth_tokens` looks like `{'model': 'authtoken.token', 'pk': '4310a8ef9e6bbce7146ef995b7dec33e0be7da02', 'fields': {'user': 3, 'created': '2023-03-10T06:22:38.138Z'}}`~~

Edit: lets just remove them

Closes #8098

## Type of change

- [x] Bug fix: non-breaking change which fixes an issue.
- [ ] New feature / Enhancement: non-breaking change which adds functionality. _Please read the important note above._
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected.
- [ ] Documentation only.
- [ ] Other. Please explain:

## Checklist:

- [ ] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by shamoon** [comment]
Actually this is still a problem because the 'pk' is still the key. Hmm

**Comment by shamoon** [comment]
Ok this should be sorted

**Comment by stumpylog** [comment]
Did that actually cause issues?  The model does define the key field as the PK: https://github.com/encode/django-rest-framework/blob/9016efe3fc412488df92912c619f8f24fed2937c/rest_framework/authtoken/models.py#L9-L18

So I would have expected Django the just handle it.

**Comment by shamoon** [comment]
No the key as pk thing is fine, it’s just that I guess because it’s the primary key the serialize method doesn’t export key in the fields so that’s the real problem here

File: src/documents/management/commands/document_exporter.py [hunk 1]
```diff
@@ -286,6 +285,20 @@ def dump(self):
                     serializers.serialize("json", manifest_key_to_object_query[key]),
                 )
 
+            # Add the auth tokens to the manifest, serialized manually
+            manifest_dict["auth_tokens"] = [
+                {
+                    "model": "authtoken.token",
+                    "pk": t.pk,
```
Comment (stumpylog) [inline src/documents/management/commands/document_exporter.py:292]: Does this then put the key out in plaintext?  I think it would

**Comment by stumpylog** [comment]
Yeah, that seems best.  Maybe drop a note in the docs for the exporter?  I would assume a user would think to re-generate keys if they're moving an install or restoring, but maybe a heads up is nice too.

**Review by stumpylog — APPROVED** [review]

**Pull request merged**